src/DependencyInjection/MercureExtension.php

```diff
@@ -67,9 +73,29 @@ public function load(array $configs, ContainerBuilder $container)
             $bus = $hub['bus'] ?? null;
             $attributes = null === $bus ? [] : ['bus' => $hub['bus']];
             $publisherDefinition->addTag('messenger.message_handler', $attributes);
+
+            if ($config['enable_profiler'] && class_exists(Stopwatch::class)) {
```

Comment:
Maybe could you store the result of this test in a variable outside of the loop?